python_modules/dagster/dagster_tests/core_tests/structured_config_tests/test_errors.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from dagster import Config, op
+from dagster._config.structured_config import ConfigurableResource
+from dagster._core.errors import DagsterInvalidPythonicConfigDefinitionError
+
+
+def test_invalid_config_type_basic() -> None:
+    class MyUnsupportedType:
+        pass
+
+    class DoSomethingConfig(Config):
+        unsupported_param: MyUnsupportedType
+
+    with pytest.raises(
+        DagsterInvalidPythonicConfigDefinitionError,
+        match="""Error defining Dagster config class <class 'test_errors.test_invalid_config_type_basic.<locals>.DoSomethingConfig'> on field 'unsupported_param'.
+Unable to resolve config type <class 'test_errors.test_invalid_config_type_basic.<locals>.MyUnsupportedType'>.
+
+
+This value can be a:
```

Comment:
[1]
maybe change "value" to something like "config field type" or "config type" to clearly indicate that it's the type of the config field that needs to be changed to something in the list?